frontend/app/routes/$lang/_public/apply/$id/adult-child/confirmation.tsx

```diff
@@ -210,6 +211,7 @@ export async function action({ context: { session }, params, request }: ActionFu
 }
 
 export default function ApplyFlowConfirm() {
+  const featureEnabled = !useFeature('view-letters');
```

Comment:
We should be displaying this new text only when the `view-letters` feature flag is enabled right?
```suggestion
  const featureEnabled = useFeature('view-letters');
```

(same comment for the other `confirmation.tsx` files)

Reply:
> We should be displaying this new text only when the `view-letters` feature flag is enabled right?
> 
> (same comment for the other `confirmation.tsx` files)

Maybe I shouldn't name the boolean featureEnabled, featuredFlaged should make more sense, since the new text is feature flaged, it will not showup until the letter is available.
I will make the change.